packages/api-reference/src/components/ApiClientModal.vue

```diff
@@ -63,9 +63,11 @@ const isMobile = useMediaQuery('(max-width: 1000px)')
             </Sidebar>
           </div>
         </template>
+        <!-- Fonts are fetched by @scalar/api-reference already, we can safely set `withoutDefaultFonts: true` -->
```

Comment:
But if api reference does not load the fonts they would be loaded through the client otherwise. :)

or we pass down the config, but it’s easier to only load them once in that context.

Reply:
Oh I see, makes sense!